apps/redditrplace/reddit_r_place.star

```diff
@@ -0,0 +1,54 @@
+"""
+Applet: Reddit R-Place
+Summary: Bits of r/place 2022
+Description: See tidbits of what Redditors created for r/place 2022.
+Author: funkfinger
+"""
+
+load("render.star", "render")
+load("http.star", "http")
+load("random.star", "random")
+load("animation.star", "animation")
+load("schema.star", "schema")
+
+HEIGHT = 32
+WIDTH = 64
+
+TILE_WIDTH = WIDTH * 3
+TILE_HEIGHT = HEIGHT * 3
+
+def main():
+    # grab a coordinate from r/place image...
+    x = -1 * (random.number(WIDTH, 2000 - TILE_WIDTH))
+    y = -1 * (random.number(HEIGHT, 2000 - TILE_HEIGHT))
+
+    # randomized which way it scrolls...
+    scrollX = x - ((random.number(0, 2 * WIDTH)) - WIDTH)
+    scrollY = y - ((random.number(0, 2 * HEIGHT)) - HEIGHT)
+
+    # get a the final r/place image
+    url = "https://i.imgur.com/rzUhL4w.png"
```

Comment:
Could this image be cached?

Reply:
For sure! Wasn't sure how the backend handles it, but as it currently exists, it's a static image. Gone ahead and cached for 30 days

Reply:
adding another pull request to incorporate the cache...